<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>G_MAIN_WINDOW</class>
 <widget class="QMainWindow" name="G_MAIN_WINDOW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1852</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eurecar Replayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1841</width>
      <height>761</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Camera</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView_cam_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>455</width>
        <height>286</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_cam_2">
      <property name="geometry">
       <rect>
        <x>464</x>
        <y>10</y>
        <width>455</width>
        <height>286</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_cam_3">
      <property name="geometry">
       <rect>
        <x>918</x>
        <y>10</y>
        <width>455</width>
        <height>286</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_cam_4">
      <property name="geometry">
       <rect>
        <x>1372</x>
        <y>10</y>
        <width>455</width>
        <height>286</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_cam_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>295</y>
        <width>455</width>
        <height>286</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_cam_6">
      <property name="geometry">
       <rect>
        <x>464</x>
        <y>295</y>
        <width>455</width>
        <height>286</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_cam_7">
      <property name="geometry">
       <rect>
        <x>918</x>
        <y>295</y>
        <width>455</width>
        <height>286</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_cam_8">
      <property name="geometry">
       <rect>
        <x>1372</x>
        <y>295</y>
        <width>455</width>
        <height>286</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>590</y>
        <width>451</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Load setting</string>
      </property>
      <widget class="QPushButton" name="pushButton_set_img_loadpath">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>set load path</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_img_loadpath">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>321</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>69</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of camera :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_cam_num">
       <property name="geometry">
        <rect>
         <x>153</x>
         <y>66</y>
         <width>51</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_start_load">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>67</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Start load</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>590</y>
        <width>451</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Save setting</string>
      </property>
      <widget class="QPushButton" name="pushButton_set_img_savepath">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>set load path</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_img_savepath">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>321</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_save_img">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>67</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>save</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>LiDAR</string>
     </attribute>
     <widget class="QVTKWidget" name="qvtkWidget_lidar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>851</width>
        <height>671</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_save_lidar_data">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>690</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>save</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+S</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_sensor_fusion">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>70</y>
        <width>960</width>
        <height>604</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_sensor_fusion">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>10</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Sensor Fusion</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1852</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
